packages/router/__tests__/index.test.ts

```diff
@@ -6,7 +6,7 @@ const router: Router = {
     router: {
       path: "/some-path",
       page: 2,
-      location: state => new URL(state.router.path)
+      location: ({ state }) => new URL(state.router.path)
```

Comment:
Does this work as expected? Because I think that `URL` will throw if the argument is not a proper url. And in this case is only the `path` part.

Reply:
Yes, you're right the code is wrong but those tests only test types, not the actual code. So that part is just a dummy implementation of a function that receives state and returns a URL, which is the thing we are interested in :)

Reply:
Oh, sorry, I missed it was a test file.

Reply:
No prob!